.github/workflows/e2e-functional-template.yml

```diff
@@ -159,6 +162,7 @@ jobs:
         env:
           PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD: 1
         run: |
+          brew install nss
```

Comment:
adding this to prevent electron crashing when test runs

Reply:
What does this do?

Reply:
on macos since we launch electron with option `--no-sandbox`. The app was crashing during execution.
I think if not directly, electron has some dependency that might need nss installed

Reply:
Interesting...